CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v0.23.0 - 2024-12-13
+
+### Bug fixes
+- Make `Waiting.resume()` idempotent [[20e5898]](https://github.com/aiidateam/plumpy/pull/285)
```

Comment:
Tend to now link to the exact commit instead of the pull request. See links in v0.22.3 release for example of URL.

BTW, I use the following bash function:
```bash
function release-notes() {
    if [ $# -eq 0 ]; then
        echo "Error: repository org and name not specified"
        return
    fi
    git log --pretty="- %s [[%h]](https://github.com/$1/commit/%H)" $(git describe --tags --abbrev=0)..HEAD
}
```
which is then called from the repos root as 
```
release-notes aiidateam/plumpy
```
and it should automatically generate the changelog lines with correct links